### ISSUE TITLE
[Exchange Refinement] Vertical Order Book left

### ISSUE THREAD

**Issue by sschiessl-bcp** [bitshares/bitshares-ui#1873]
Bring back the "Pin Order Book" icon for the new exchange layout that resembles the old one. This allows to have a feeling of the old layout while users may want to explore the new "Personalize" option.

**Comment by kapeer42** [comment]
Not sure if this deserve a new issue:
On current develop, in the Personalize modal, you have to click on "Horiz. (...)" to get the vertical order book, I would expect it to be the other way around.

**Comment by startailcoon** [comment]
I think what @sschiessl-bcp refers to is the layout called "Vert. Orderbook, Original" which was added to resemble the old layout we had for old users. It lacks the option to "pin" the order book to the left though.

As I've described in my issue #1863 , I write about this as well.

**Comment by sschiessl-bcp** [comment]
> I think what @sschiessl-bcp refers to is the layout called "Vert. Orderbook, Original" which was added to resemble the old layout we had for old users

Yes exactly. Introduce the new layouting options via "Personalize" while having minimal friction through providing something very similar to old one

**Comment by startailcoon** [comment]
This will be fixed and pushed to the Branch of #1477.

Requested hours: 1 hour

![image](https://user-images.githubusercontent.com/12114550/45930101-ae2e5000-bf5b-11e8-89a1-c99e69b6093b.png)

**Comment by startailcoon** [comment]
This is done and required 2 hours of work to complete. Most work related to SCSS styling and dynamic canvas sizing optimisations.
